layouts/index.html

```diff
@@ -98,6 +95,18 @@ <h2 class="title--answer">Yes, we are <a href="https://wayland.freedesktop.org/"
         <a href="https://github.com/Zeioth/wofi-emoji">wofi-emoji</a>,
         <a href="https://github.com/kenshaw/wofimoji">wofimoji</a>
       </li>
+      <li class="list__item--ok">
+        Emulators:
+        <a href="https://waydro.id">Android>Waydroid</a>,
```

Comment:
I don't think we need the `Platform>Software` label everywhere, also it's inconsistent among all items in terms of spacing. 

Just the emulator name alone should be sufficient.